### DISCUSSION TITLE
Separate option to also erradicate 'Groups' feed on Facebook?

### DISCUSSION THREAD

**Discussion by Darkmaster006** [jordwest/news-feed-eradicator#413] [Ideas]
I was wondering if there was a way to erradicate also the 'Groups' feed on Facebook (https://www.facebook.com/groups/feed/), as a stand-alone option so that one could leave it erradicated or not while at the same time having the 'main' feed erradicated (or not). Hopefully this could be implemented. Thank you very much for all the work you do, this add-on is really helpful.

**Comment by theDanielJLewis** [comment]
Eek. Now, the groups page is being completely eradicated, and I don't like that. I want only the main feed eradicated. But I go to the groups page for content I want to actually engage with.

**Comment by Darkmaster006** [comment]
> Eek. Now, the groups page is being completely eradicated, and I don't like that. I want only the main feed eradicated. But I go to the groups page for content I want to actually engage with.

I wanted the same, actually! As I said in my post, the best thing is to have the option to either erradicate it or not, separate from the feed.

**Comment by Darkmaster006** [comment]
> Eek. Now, the groups page is being completely eradicated, and I don't like that. I want only the main feed eradicated. But I go to the groups page for content I want to actually engage with.

Nevertheless, if you need a solution right now, I think that, with uBlock, you can block the feed with a personalised filter, just use the "block element" tool to try and see.
